packages/contracts/contracts/interfaces/IChugSplashRegistry.sol

```diff
@@ -1,12 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.9;
-
-import { IChugSplashManager } from "./IChugSplashManager.sol";
-
-/**
- * @title ChugSplashRegistry
- * @notice Interface that must be inherited the ChugSplash registry.
- */
-interface IChugSplashRegistry {
-    function projects(address, bytes32) external returns (IChugSplashManager);
-}
```

Comment:
The `projects` function here didn't match the `ChugSplashRegistry` (different return values). I didn't see a point in keeping this file.